.gitignore

```diff
@@ -170,3 +170,8 @@ Xcode/
 
 # gdb
 .gdb_history
+*.ogg
```

Comment:
```suggestion
```

Reply:
This and more evidently got included by mistake, sorry.